tests/rmutex/main.c

```diff
@@ -0,0 +1,91 @@
+/*
+ * Copyright (C) 2016 Freie Universität Berlin
```

Comment:
@melshuber what @kaspar030 meant was: you are not working for FUB so you can put your own name (or the one of your company) there ;-)